src/lxc/seccomp.c

```diff
@@ -791,6 +791,17 @@ int lxc_seccomp_load(struct lxc_conf *conf)
 		ERROR("Error loading the seccomp policy: %s.", strerror(-ret));
 		return -1;
 	}
+
+/* After load seccomp filter into the kernel successfully, export the current seccomp
+ * filter to log file */
+#if HAVE_SCMP_FILTER_CTX
+	if (lxc_log_get_level() <= LXC_LOG_PRIORITY_TRACE && lxc_log_fd >= 0) {
```

Comment:
There's unfortunately one more thing I need to annoy you with. It isn't sufficient to check for `lxc_log_get_level()`. Can you also add a check for `conf->loglevel <= LXC_LOG_PRIORITY_TRACE`? That would excellent!